app/grants/templates/grants/cart/eth.html

```diff
@@ -3,408 +3,260 @@
+                    Please allow popups from https://gitcoin.co in your browser settings.
+                  </div>
                 </div>
-              </grants-cart-ethereum-zksync>
-              {% comment %} End HTML Template for cart-ethereum-zksync.js {% endcomment %}
               </div>
-            <div class="col-auto">
-              <button v-if="!isZkSyncDown" class="btn btn-outline-gc-blue shadow-none py-3 mt-1" id='js-fundGrants-button'
-                @click="standardCheckout" :disabled="isCheckoutOngoing || grantsByTenant.length > maxCartItems" 
-                v-b-tooltip.hover.top="'Traditional Ethereum easy payout using funds from your connected wallet (easiest option).'"
-              >
-                Standard Checkout
-              </button>
-              <button v-else class="btn btn-gc-blue button--full shadow-none py-3 mt-1" id='js-fundGrants-button'
-                @click="standardCheckout" :disabled="isCheckoutOngoing || grantsByTenant.length > maxCartItems" 
-                v-b-tooltip.hover.top="'Traditional Ethereum easy payout using funds from your connected wallet (easiest option).'"
-              >
-                Standard Checkout
-              </button>
-            </div>
-          </div>
-          <p v-if="grantsByTenant.length > maxCartItems" class="font-smaller-1 mt-2">
-            Cart must have a maximum of [[maxCartItems]] items, but has [[ grantsByTenant.length ]] items.
-            Please remove [[ grantsByTenant.length - maxCartItems ]] item<span v-if="grantsByTenant.length - maxCartItems > 1">s</span>
-            from your cart to continue checkout.
-          </p>
-          <p v-else-if="isZkSyncDown" class="font-smaller-1 mt-2">
-            zkSync is down at the moment, so if you'd like to checkout now you'll have to use standard checkout!
-          </p>
+            </template>
+          </b-modal>
         </div>
-      </div>
+      </grants-cart-ethereum-zksync>
+    
+      <button class="btn btn-primary py-2 ml-2 mr-0" id='js-fundGrants-button'
+        @click="standardCheckout" :disabled="isCheckoutOngoing || grantsByTenant.length > maxCartItems"
+        v-b-tooltip.hover.top="'Traditional Ethereum easy payout using funds from your connected wallet (easiest option).'"
+      >
+        Standard Checkout
+      </button>
     </div>
   </div>
+
+  <small class="row justify-content-end mt-2">
+    <div v-if="grantsByTenant.length > maxCartItems">
+    Cart must have a maximum of [[maxCartItems]] items, but has [[ grantsByTenant.length ]] items.
+    Please remove [[ grantsByTenant.length - maxCartItems ]] item<span v-if="grantsByTenant.length - maxCartItems > 1">s</span>
+    from your cart to continue checkout.
+    </div>
+    <div v-else-if="isZkSyncDown">
+      zkSync is down at the moment, so if you'd like to checkout now you may need to use standard checkout
+    </div>
+  </small>
+
+  <!-- Add some empty space before the footer -->
+  <div class="m-5"></div>
```

Comment:
I'm assuming these changes are restricted to ETH in checkout and will not affect the other cross chain tabs

Reply:
Correct